src/kernel_um.h

```diff
@@ -48,6 +48,8 @@ CXPLAT_EXTERN_C_BEGIN
 #define STATUS_GENERIC_COMMAND_FAILED ((NTSTATUS)0xC0150026L)
 #define STATUS_CONTENT_BLOCKED ((NTSTATUS)0xC0000804L)
 #define STATUS_RESOURCE_NOT_OWNED ((NTSTATUS)0xC0000264L)
+#define STATUS_INVALID_DEVICE_STATE ((NTSTATUS)0xC0000184L)
```

Comment:
Move up into numerical order.  (Looks like above lines are already out of order, which should also be fixed, but ok to do in this PR or in a separate PR.)